bin/migrate-oats-data/submissions/__init__.py

```diff
@@ -0,0 +1 @@
+from .app_submissions import *
```

Comment:
Import only those items that are supposed to be accessed from outside the module. In this case, there are only
"process_alcs_app_submissions" and "clean_application_submission" so it should be the only one in import. Otherwise, it is possible to call "get_update_query_for_other" anywhere the submission module is imported.